src/client.rs

```diff
@@ -114,9 +114,13 @@ impl TClientField for RectClientField {
             }
         } else if is_mouse_button_released(MouseButton::Left) {
             if !is_mouse_button_down(MouseButton::Right) {
-                Some(vec![Coords::RectCoords { row, col }]) // todo: add neighbours
+                if self.rows[row][col] == VisibleCellState::Closed {
```

Comment:
logic in classical minesweeper here is like this: if u releae left when right is pressed, u open `all closed cells` surrounding the one was clicked and itself, IF the clicked one is open and has sufficient ammount of marked neighbours (for example, if u click on opened 3, all the neighbours will open if 3 of them are marked), otherwise nothing happens, but the animation